lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -116,7 +116,7 @@ sub list {
     #         0;
     #     }
     # }
-    my @scheduled = $scheduled->search({})->all;
+    my @scheduled = $scheduled->search({blocked_by_id => undef})->all;
```

Comment:
This needs to pass all the scheduled jobs, or they won't be highlighted

Reply:
Yes. That the blocked-by-ones are missing in the table was actually an issue I've got complaints about (when the new scheduler was deployed).

Reply:
Fixed (and hotpatched)